### ISSUE TITLE
Detect subdomains

### ISSUE THREAD

**Issue by peaceful-james** [Betree/burnex#45]
Validating top-level domains works fine:
```
Burnex.is_burner?("hello@mailinator.com")
true
```

But subdomains are not validated correctly:
```
Burnex.is_burner?("hello@mail2.mailinator.com")
false
```

Shouldn't subdomains be validated too?

**Comment by peaceful-james** [comment]
I made a PR for this: https://github.com/Betree/burnex/pull/46

**Comment by peaceful-james** [comment]
this is fixed in 2.1.0 now
https://github.com/Betree/burnex/pull/46
